src/raftstore/store/peer.rs

```diff
@@ -199,6 +200,7 @@ pub struct Peer {
     pending_reads: ReadIndexQueue,
     // Record the last instant of each peer's heartbeat response.
     pub peer_heartbeats: FlatMap<u64, Instant>,
+    pub learner_heartbeats: FlatMap<u64, Instant>,
```

Comment:
Seem we can use peer_heartbeats only.

Reply:
In my opinion, `peer` means it can vote so that `learner` is not `peer`. So considering function [check_peers](https://github.com/hicqu/tikv/blob/3ebc84ee7b761bce7a27c782847c030e627f774c/src/raftstore/store/peer.rs#L548), I think seperate it is better.

Reply:
Maybe rename `peer_heartbeats` to `follower_heartbeats` is better?